docs/cloud/pouta/heat-orchestration.md

```diff
@@ -64,45 +71,167 @@ stacks:
+heat_template_version: queens
+
+description: Resource Group to deploy one or several instance(s)
+
+parameters: # Parameter retrieves from heat_params.yaml file.
+  count:
+    description: Number of resources
+    type: string
+
+resources:
+  instances_group:
+    type: OS::Heat::ResourceGroup
+    properties:
+      count: { get_param: count }
+      resource_def:
+        type: heat_stack_vm.yaml # We define our previous template for the type.
         properties:
-          image: CentOS-7
-          flavor: standard.small
-          key_name: my-key
-    outputs:
-        server_networks:
-          description: >
-            Outputs the networks of the
-            deployed server
-          value: { get_attr: [instance0, networks] }
-
-  [Template versions]: https://docs.openstack.org/heat/latest/template_guide/hot_spec.html#heat-template-version
-  [OpenStack developer Guide]: http://docs.openstack.org/developer/heat/template_guide/
-  [Heat wiki]: https://wiki.openstack.org/wiki/Heat
-  [command line reference]: http://docs.openstack.org/cli-reference/openstack.html
-  [Heat CLI reference]: https://docs.openstack.org/mitaka/cli-reference/heat.html
+          vm_name: test-stack-%index% # Value %index% will be increased if more than one vm is created.
+
+outputs:
+  print_out:
+    value: { get_attr: [instances_group, network] }
+```
+
+And finally, we'll create an ansible script that will build and deploy our vm(s). Let's call it `build-heat-stack.yaml`:  
+
+```yaml
+- hosts: localhost
+  gather_facts: false
+  vars:
+    heat_environment_file: "heat_params.yaml" # Be sure that the file is located at the same level as build-heat-stack.yaml
+
+  tasks:
+    - name: Build a Heat stack VM
+      register: heat_stack
+      os_stack:
+        name: "{{ stack_name }}"
+        state: present
+        template: "servers_group.yaml" # Be sure that the file is located at the same level as build-heat-stack.yaml
+        environment: 
+          - "{{ heat_environment_file }}"
+
+    - name: Print out network
+      debug:
+        var: heat_stack
+```
+
+Source your cPouta project (`source project_xxxxx.sh`) and run the command:   
+```sh
+ansible-playbook -e stack_name="test-stack" build-heat-stack.yaml`
+```
+
+If everything went well, you can check that the stack was created, either using the web interface of cPouta or typing the command: `openstack stack list`.  
+You can also check the instance(s) created: `openstack server list`.  
+
+If you delete the stack, all the resources created from it will be deleted as well.
+
+### Heat guidelines and command references
+
+For more information, visit the OpenStack [Heat wiki](https://wiki.openstack.org/wiki/Heat). For a full reference to the OpenStack command line client, see the [command line reference](http://docs.openstack.org/cli-reference/openstack.html).  
+Here is an example on our GitHub to deploy one or several instances with nginx: (https://github.com/CSCfi/heat-openstack-example)
```

Comment:
```suggestion
Here is an example on our GitHub to deploy one or several instances with nginx: (<https://github.com/CSCfi/heat-openstack-example>)
```

Reply:
I changed to `[Read more about the GitHub repo](https://github.com/CSCfi/heat-openstack-example){ target="_blank" }` according the documentation: https://github.com/CSCfi/csc-user-guide/blob/master/FAQ.md#how-to-add-links